docs/src/devdocs/interpolations.md

```diff
@@ -1,12 +1,47 @@
 # [Interpolations](@id devdocs-interpolations)
 
-## Type definitions
+All `Interpolation`s should subtype `Interpolation{shape, order}`,
+where `shape <: AbstractRefShape` is the reference shape for which
+the interpolation is defined and [`order`](@ref Ferrite.getorder) is the characteristic interpolation
+order. The [how-to at bottom of this page](@ref devdocs-howto_new-interpolation) describes how to implement a new interpolation.
 
-Interpolations are subtypes of `Interpolation{shape, order}`, i.e. they are
-parametrized by the reference element and its characteristic order.
+## Methods to be implemented for a new interpolation
+### Always
+```@docs
+Ferrite.reference_shape_value(::Interpolation, ::Vec, ::Int)
+Ferrite.reference_coordinates(::Interpolation)
+Ferrite.vertexdof_indices(::Interpolation)
+Ferrite.facedof_indices(::Interpolation)
+Ferrite.facedof_interior_indices(::Interpolation)
+Ferrite.edgedof_indices(::Interpolation)
+Ferrite.edgedof_interior_indices(::Interpolation)
+Ferrite.volumedof_interior_indices(::Interpolation)
+Ferrite.getnbasefunctions(::Interpolation)
+Ferrite.adjust_dofs_during_distribution(::Interpolation)
+```
 
-### Fallback methods applicable for all subtypes of `Interpolation`
+### For special interpolations
+#### Discontinuous interpolations
+For discontinuous interpolations, implementing the following methods might be required.
```

Comment:
```suggestion
For discontinuous interpolations, implementing the following methods might be required to apply Dirichlet boundary conditions.
```

We should also add the exact conditions here under which we can apply Dirichlet in first place.

Reply:
No concrete text yet. Essentially right now these only work if we have
1. Nodal ansatz
2. We have $N_i(x) = 0$ for all ansatz functions which are not on the face

Reply:
Actually, the nodal ansatz is not required if using `L2ProjectedDirichlet`, which still uses e.g. `dirichlet_facedof_indices`.

To avoid stalling on this addition, could we discuss and add this in a separate PR?

Reply:
Yea, let us open an issue to track and discuss these ideas first

Reply:
#1186